content/docs/18-knowledgebase/00-how-to/2-palette-sso-with-adfs.md

```diff
@@ -0,0 +1,222 @@
+
+Open a web browser and navigate to your Palette subscription. Navigate to **Tenant Settings** --> **Users & Teams** --> **Teams** tab. Click the **+ Create Team** button.
+
+![Create Palette Team](palette-sso-with-adfs-images/21-create_team.png)
+
+
+Specify `Admins` in the **Team name** field. We don't need to set any members now, as this will happen automatically from the SSO. Click **Confirm** to create the team.
+
+![Name Palette Team](palette-sso-with-adfs-images/22-name_team.png)
+
+
+You'll be returned back to the list of Teams. Click on the newly created `Admins` Team to open its details screen. To give this team administrative access to the entire tenant (and thus all Projects in it), we will give this Team the `Tenant Admin` role. Click on **Tenant Roles** --> **+ Add Tenant Role**:
+
+![Palette Tenant Roles](palette-sso-with-adfs-images/23-tenant_roles.png)
+
+
+Click on **Tenant Admin** to enable the role, this automatically enables all other roles with fewer permissions. Click **Confirm** to add the role:
+
+![Add Tenant Role](palette-sso-with-adfs-images/24-add_tenant_role.png)
+
+
+You will see the `Roles have been updated` message to confirm the change. Repeat this procedure for any other teams you need, ensuring they're given the appropriate permissions. Finally click the `X` next to **Team Details** in the top left corner to exit this screen.
+
+Congratulations! You have now successfully configured Palette SSO based on OIDC with Microsoft AD FS!
+
+
+# Validation
+Let's log in through SSO as a user that is a member of the `SpectroTeam - Admins` group in Active Directory to verify that we're automatically getting added to the `Admins` group in Palette.
+
+If you're still logged into Palette with a non-SSO user, log out first by clicking on your name at the top right and selecting `Logout from <tenant name>`:
+
+![User Logout](palette-sso-with-adfs-images/25-user_logout.png)
+
+
+Notice that the Palette login screen now only shows a big **Sign in** button and no longer a username/password field. Below this button there is an **SSO issues? --> Use your password** link. This can be used to bypass SSO and login with a local Palette account in case there's ever a problem with SSO and you need to access Palette without SSO.
+
+Click the **Sign in** button to login via SSO:
+
+![User SSO Login](palette-sso-with-adfs-images/26-palette_login.png)
+
+
+If this is the first time, you will be redirected to the Microsoft AD FS login page. Depending on your company's SSO settings, this could be a simple login form or also require MFA (Multi-Factor Authentication). Make sure you log in as a user that is a member of the `SpectroTeam - Admins` group in Active Directory. Once authenticated, you will automatically be redirected back to Palette and logged into Palette as that user.
```

Comment:
```suggestion
If this is the first time you are login in with SSO, you will be redirected to the Microsoft AD FS login page. Depending on your organization's SSO settings, this could be a simple login form or require MFA (Multi-Factor Authentication).

Make sure you log in as a user that is a member of the `SpectroTeam - Admins` group in Active Directory. Once authenticated, you will automatically be redirected back to Palette and logged into Palette as that user.
```